interfaces/cython/cantera/test/test_transport.py

```diff
@@ -33,7 +33,7 @@ def test_mixtureAveraged(self):
         self.assertArrayNear(Dbin1, Dbin1.T)
 
     def test_multiComponent(self):
-        with self.assertRaises(Exception):
+        with self.assertRaises(AttributeError):
```

Comment:
Actually, I think the test here is wrong. The thing that we meant to test there is that `multi_diff_coeffs` raises an error from Cantera if we are not using a multicomponent model, but of course the typo in the property name also raises `Exception`, so this is a good example of why it was good to add the `CanteraError` Python class.